cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```

Comment:
You can get rid of this examples. I was impressed at first that you have written so many cases ;)